src/brs/http/GetSubscription.java

```diff
@@ -42,7 +47,9 @@ JsonElement processRequest(HttpServletRequest req) {
       response.addProperty(ERROR_DESCRIPTION_RESPONSE, "Subscription not found");
       return response;
     }
+    // retrieve additional transaction data
```

Comment:
seems like an not-so-useful comment

Reply:
Yes, maybe it was more like a thought which i had in that case ;-) i will remove it.

Reply:
done